src/ApparentHorizons/StrahlkorperGr.hpp

```diff
@@ -241,4 +244,38 @@ double dimensionful_spin_magnitude(
     const tnsr::ii<DataVector, 3, Frame>& spatial_metric,
     const StrahlkorperTags::StrahlkorperTags_detail::Jacobian<Frame>& tangents,
     const YlmSpherepack& ylm, const Scalar<DataVector>& area_element) noexcept;
+
+/*!
+ * \ingroup SurfacesGroup
+ * \brief Spin vector of a 2D `Strahlkorper`.
+ *
+ * \details Computes the spin vector of a `Strahlkorper` in a `Frame`, such as
+ * `Frame::Inertial`. The result is a `std::array<double, 3>` containing the
+ * Cartesian components of the spin vector, whose (dimensionful or
+ * dimensionless) magnitude is `spin_magnitude`. The spin vector is given by
```

Comment:
This confused me; it is less clear than the comment in the .cpp which just states that `spin_vector` inherits the normalization of `spin_magnitude`? I would remove the parenthetical and add a new sentence to make this clearer.